docs/topics/system-requirements.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * docs/getting-started-guide/master.adoc
+
```

Comment:
@sbeskin-redhat @stoobie  I am a little hesitant about calling this section "reference" because it's something we want every user to read and not gloss over as "technical background." But you know our users better than me. What do you think?

Reply:
@RichardHoch @sbeskin-redhat I honestly never considered whether Prerequisites are reference or not. I'm going to defer to the RHV Planning and Preferequisites guide and says that this is REFERENCE.